bin/options.js

```diff
@@ -1,62 +1,30 @@
 // ShareJS options
 module.exports = {
 	// Port to listen on
-	port: 8000,
+	port: 9000,
 
 	// Database options
 	db: {
-		// DB type. Options are 'redis', 'couchdb' or 'none'. 'redis' requires the
-		// redis npm package.
+		// DB type. Options are 'postgres' or 'none'.
     //
     // If you don't want a database, you can also say db: null. With no database,
     // all documents are deleted when the server restarts.
+    type: 'pg',
```

Comment:
One thing worth noting which is not immediately obvious from this change, is that now the tests will run against postgres (previously they were not).

```
✔ db - pg - create doesnt send an error when a document is created
✔ db - pg - create sends an error if the document already exists
✔ db - pg - Calling create multiple times only results in one successful create
✔ db - pg - getSnapshot sends an error for a nonexistant doc
✔ db - pg - getSnapshot has the type, version and snapshot set when a doc is created
✔ db - pg - getSnapshot is updated after calling writeSnapshot()
✔ db - pg - getSnapshot is updated after calling writeSnapshot() with dbMeta
✔ db - pg - getSnapshot returns dbMeta through the callback
✔ db - pg - writeOp writes ops to the DB
✔ db - pg - snapshot is updated when a second op is applied
✔ db - pg - delete a non-existant document sends an error to its callback
✔ db - pg - delete deletes a document
✔ db - pg - delete with dbMeta set still works
✔ db - pg - Calling delete multiple times only results in one successful delete
✔ db - pg - delete with no callback doesnt crash
✔ db - pg - getOps returns [] for a nonexistant document, with any arguments
✔ db - pg - getOps returns [] for a new document, with any arguments
✔ db - pg - getOps returns ops
```

etc